### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by olhry**
- [DEMO LINK]
 https://olhry.github.io/layout_product-cards/
- [TEST REPORT LINK]
- (https://olhry.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls try to make all tests pass and request the review again)

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,60 @@
...
+  <body class="body">
+    <div class="card" data-qa="card">
+
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+      >
+
+      <div class="card__title">
```
Comment (agniya-i) [inline src/index.html:26]: Use semantic tags
```suggestion
      <h2 class="card__title">
```

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,60 @@
...
+        APPLE A1419 iMac 27" Retina
+        <br> 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">
```
Comment (agniya-i) [inline src/index.html:30]:
```suggestion
      <p class="card__code">
```

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,60 @@
...
+
+        <p class="card__review">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price price">
+        <div class="price__text">Price:</div>
+
+        <div class="price__value">$2,199</div>
```
Comment (agniya-i) [inline src/index.html:51]:
```suggestion
        <p class="price__text">Price:</p>

        <p class="price__value">$2,199</p>
```

File: src/index.html [hunk 4]
```diff
@@ -5,9 +5,60 @@
...
+      </div>
+
+      <a
+      href="#"
+      class="card__button"
+      data-qa="hover"
+      >
```
Comment (agniya-i) [inline src/index.html:58]:
```suggestion
      <a
        href="#"
        class="card__button"
        data-qa="hover"
      >
```

There should be 2-space indentation for attributes

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,122 @@
...
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+}
+
+.body {
+  margin: 0;
+}
+
+.card {
```
Comment (agniya-i) [inline src/style.css:10]: You should create separate file for each BEM block (Checklist p11)

So create separate files for `card`, `stars` etc

**Review by agniya-i — CHANGES_REQUESTED** [review]

File: src/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```
Comment (Anastasiia-Svintsova) [inline src/blocks/card.scss:4]: Move the box-sizing to the * selector to ensure a consistent box model throughout your page

File: src/blocks/card.scss [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+
+  &__comments {
+    font-weight: 400;
+    font-size: 10px;
+    color: #060b35;
```
Comment (Anastasiia-Svintsova) [inline src/blocks/card.scss:43]: You use this color not once. Move repeatable values to variables

File: src/blocks/stars.scss [hunk 8]
```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  // margin: 0;
```
Comment (Anastasiia-Svintsova) [inline src/blocks/stars.scss:3]: Remove redundant commented code

File: src/index.html [hunk 9]
```diff
@@ -5,9 +5,59 @@
...
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br> 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:30]: Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
      </h2>
      
      <p class="card__code">
```

File: src/index.html [hunk 10]
```diff
@@ -5,9 +5,59 @@
...
+
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br> 5K Monoblock (MNED2UA/A)
```
Comment (Anastasiia-Svintsova) [inline src/index.html:28]: The `br` tag is redundant here. Just use the correct width for this section

File: src/index.html [hunk 11]
```diff
@@ -5,9 +5,59 @@
...
+      </h2>
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="rating">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:34]:
```suggestion
      <div class="card__rating">
```

File: src/blocks/card.scss (hunk 7 above)
Comment (olhry) [reply src/blocks/card.scss:43]: переніс в утіліти

File: src/index.html (hunk 9 above)
Comment (olhry) [reply src/index.html:30]: виправив

File: src/index.html (hunk 10 above)
Comment (olhry) [reply src/index.html:28]: видалив

File: src/blocks/card.scss (hunk 6 above)
Comment (olhry) [reply src/blocks/card.scss]: Виконав

File: src/blocks/card.scss [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    height: 14px;
+    color: #616070;
+    margin: 0 0 16px;
+  }
+}
+
+.rating {
```
[the remark this answers is not in the document]
Comment (olhry) [reply src/blocks/card.scss]: В моєму розумінні я зробив 2 ВЕМ блоки, але якщо я зробив неправильно, допоможіть будь-ласка, як їх побудувати правильно.

File: src/blocks/stars.scss (hunk 8 above)
Comment (olhry) [reply src/blocks/stars.scss]: видалив

File: src/index.html [hunk 13]
```diff
@@ -5,9 +5,59 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__comments">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="price">
```
Comment (lerastarynets) [inline src/index.html:48]:
```suggestion
      <div class="card_price">
```

File: src/index.html [hunk 14]
```diff
@@ -5,9 +5,59 @@
...
+        <p class="price__text">Price:</p>
```
Comment (lerastarynets) [inline src/index.html:49]:
```suggestion
        <p class="card__text">Price:</p>
```

File: src/index.html [hunk 15]
```diff
@@ -5,9 +5,59 @@
...
+
+        <p class="price__value">$2,199</p>
```
Comment (lerastarynets) [inline src/index.html:51]:
```suggestion
        <p class="card__value">$2,199</p>
```

File: src/index.html [hunk 16]
```diff
@@ -5,9 +5,59 @@
...
+      </div>
+
+      <a
+        href="#"
+        class="button"
```
Comment (lerastarynets) [inline src/index.html:56]:
```suggestion
        class="card_button"
```

File: src/blocks/card.scss (hunk 12 above)
[the remark this answers is not in the document]
Comment (lerastarynets) [reply src/blocks/card.scss]: @olhry left comments on how to modify your bem classes so as you have only 2 BEM blocks (card and stars)

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Almost there) Just several more things needs to be fixed

File: src/index.html (hunk 13 above)
Comment (olhry) [reply src/index.html:48]: done

File: src/index.html (hunk 14 above)
Comment (olhry) [reply src/index.html:49]: done

File: src/index.html (hunk 15 above)
Comment (olhry) [reply src/index.html:51]: done

File: src/index.html (hunk 16 above)
Comment (olhry) [reply src/index.html:56]: done

**Review by maxim2310 — APPROVED** [review]
Nice Work!!!

**Pull request closed without merging**